lib/utils.js

```diff
@@ -86,6 +86,10 @@ module.exports = {
             return schemaRetriever.getSchemaAndNames(type, schemaType, name).then((schema) => module.exports.splitArgsForSchema(schema, schemaType, false));
     },
 
+    getMixin(schemaRetriever, kind) {
+        return schemaRetriever.getMixins(kind);
+    },
+
```

Comment:
do you need this indirection?

Reply:
just trying to be consistent with getschema, is it bad?

Reply:
It's extra code that does not add much to readability. So yeah I'd get rid of it.